playbooks/configure-apt-sources.yml

```diff
@@ -151,16 +185,12 @@
         - apt_artifact_found | bool
         - apt_artifact_enabled | bool
 
-    - name: Remove extra sources
-      lineinfile:
```

Comment:
Are we no longer needing to remove these lines from the default sources, even in strict mode?

Reply:
In the line above, the entire file is being replaced. This task does nothing.